test/extended/images/mongodb_replica.go

```diff
@@ -0,0 +1,126 @@
+package images
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/db"
+)
+
+var _ = g.Describe("[images][mongodb] openshift mongodb replication", func() {
+	defer g.GinkgoRecover()
+
+	const (
+		templatePath         = "https://raw.githubusercontent.com/openshift/mongodb/master/2.4/examples/replica/mongodb-clustered.json"
+		deploymentConfigName = "mongodb"
+		insertCmd            = `db.bar.save({ "status": "passed" })`
+	)
+
+	const (
+		expectedReplicasAfterDeployment = 3
+		expectedReplicasAfterScalingUp  = expectedReplicasAfterDeployment + 2
+	)
+
+	oc := exutil.NewCLI("mongodb-replica", exutil.KubeConfigPath()).Verbose()
+
+	g.Describe("creating from a template", func() {
```

Comment:
can't we even knock out that second level of Describes?
Describe("openshift mongodb replication")
  It("should create from a template")
  It("should write and read data")
......

i have to admit i'm not fluent in ginkgo best practices.